recipes/croaring/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "CRoaring" %}
+{% set version = "0.2.59" %}
+{% set sha256 = "02f80850e856001dbcd2571d140bab3d2bfb1e5bcc1e3b271d4c4e964dd65518" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/RoaringBitmap/CRoaring/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
```

(Sorry I missed this last time I looked.) Am I correct in thinking this is a shared library therefore anything that builds using it will also need it at runtime?

If so, it [should have a `run_export`](https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html?highlight=selectors#export-runtime-requirements) section. The `x.x.x` section should be changed depending on the expected ABI stability of the shared libraries:
```suggestion
  skip: true  # [win]
  run_exports:
    - {{ pin_subpackage('croaring', max_pin='x.x.x') }}
```

---

thanks @chrisburr, I committed your suggestion